### DISCUSSION TITLE
[Feature Request] Add the "customize toolbar" and "More troubleshooting information" buttons to the toolbar and the "customize toolbar" page.

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Floorp-Projects/Floorp#402] [Feature Request]
### Is your feature request related to a problem?

Both the "customize toolbar" and "More troubleshooting information" menu buttons are hidden away in the hamburger menu, requiring too much time and too many clicks to open.

### Describe the solution you'd like

Shortcuts to both buttons should be added to the "customize toolbar" page so they can be added to the URL or bookmark bars.

**Comment by menndouyukkuri** [comment]
Although it is an outsider, I don't think the two are commonly used items, and I think that "customize toolbar" can be displayed by right-clicking the toolbar, so it is not necessary to use the menu.
